src/python/grpcio/grpc/__init__.py

```diff
@@ -421,29 +423,50 @@ def __call__(self, context, callback):
 class ServerCredentials(object):
   """A value encapsulating the data required to open a secure port on a Server.
 
-  This class has no supported interface - it exists to define the type of its
-  instances and its instances exist to be passed to other functions.
+  This class has no supported interface - not even construction. It exists to
+  define the type of its instances and its instances exist to be passed to
+  other functions.
   """
 
   def __init__(self, credentials):
     self._credentials = credentials
 
 
+#################################  Parent  #####################################
+
+
+class Parent(object):
+  """An object used to chain RPCs.
```

Comment:
C# allows to specify "ContextPropagationOptions" where you decide what you want to propagate (deadline, cancellation, census stuff will be added later).